### ISSUE TITLE
Create language dropdown

### ISSUE THREAD

**Issue by discombobulateme** [saucelabs/elemental-next#379]

**Comment by aquaomarine** [comment]
@Globnartheexquisite Is this the final design for the drop-down menus?

**Comment by Globnartheexquisite** [comment]
@aquaomarine 

no, those are placeholder lookalikes. The final versions are almost done being automated.

**Comment by Globnartheexquisite** [comment]
@aquaomarine I have updated the existing models in the figma. If you look at the same model as before, they're the final versions!

**Comment by aquaomarine** [comment]
@Globnartheexquisite this?

![Image](https://user-images.githubusercontent.com/45857126/229624310-3948ab81-a4b5-4c39-9202-e2ae32879dcd.png)

**Comment by diemol** [comment]
We are now using the Dropdown provided by Infima, the CSS toolkit embedded in Docusaurus.
